dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1296,6 +1296,29 @@ def test_filters_v0(tmpdir, write_engine, read_engine):
     assert len(ddf2) > 0
 
 
+def test_fiters_file_list(tmpdir, engine):
```

Comment:
Do we need to test the list of files case as well?

Reply:
Are you suggesting we use glob to pass in a list instead of passing `"*.parquet"` directly? If so, that works for me.

Reply:
Oh I misread. This looks good. I guess it's the single file case that is missing. Where instead of a dir it's just a file

Reply:
Right - I didn't add a dedicated test for a singe file.  However,  did add a check at the end of this test that we can read in the first partition as a single file (and appropriately filter out its content).

Reply:
:facepalm: so you did. Sorry for the noise.